server/database/models.py

```diff
@@ -2,19 +2,27 @@
 from flask_bcrypt import generate_password_hash, check_password_hash
 
 class Contact(db.Document):
-    name = db.StringField(required=True, unique=True)
-    email = db.EmailField(required=True, unique=True)
+    name = db.StringField(required=True)
```

Comment:
Does this change get rid of the error? @amy-lei. I just wanted to make sure

Reply:
I havent been able to test the contact api endpoints. I've been stuck at register because of mongo kept complaining about non unique fullnames